src/main/steps/urls.ts

```diff
@@ -17,6 +17,9 @@ export const RELATIONSHIP_NOT_BROKEN_URL: PageLink = '/relationship-not-broken';
 export const HELP_WITH_YOUR_FEE_URL: PageLink = '/help-with-your-fee';
 export const HELP_PAYING_HAVE_YOU_APPLIED: PageLink = '/have-you-applied-for-help-with-fees';
 export const HELP_PAYING_NEED_TO_APPLY: PageLink = '/apply-for-help-with-fees';
+export const IN_THE_UK: PageLink = '/in-the-UK';
```

Comment:
Looks like this will need to be lower case to match the step `src/main/steps/sequence/in-the-uk/content.ts`

Reply:
surprised it worked locally

Reply:
might be better to change the folder to captial UK or covert urls to lowercase when finding the file

Reply:
i guess the convention is lowercase urls, ill change it to lowercase 👍